src/main/kotlin/no/nav/tilleggsstonader/klage/felles/util/TekstUtil.kt

```diff
@@ -3,12 +3,13 @@ package no.nav.tilleggsstonader.klage.felles.util
 import java.time.LocalDate
 import java.time.LocalDateTime
 import java.time.format.DateTimeFormatter
+import java.util.*
 
 object TekstUtil {
     fun String.storForbokstav() = this.lowercase().replaceFirstChar { if (it.isLowerCase()) it.uppercase() else it.toString() }
 
     object DatoFormat {
-        val DATE_FORMAT_NORSK: DateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy")
+        val DATE_FORMAT_NORSK: DateTimeFormatter = DateTimeFormatter.ofPattern("d.MMMM.yyyy", Locale.forLanguageTag("nb-NO"))
```

Comment:
Tror ikke det skal være "." etter måned
```suggestion
        val DATE_FORMAT_NORSK: DateTimeFormatter = DateTimeFormatter.ofPattern("d.MMMM yyyy", Locale.forLanguageTag("nb-NO"))
```

Reply:
Det har du helt rett i, bra spotta 🙌